### PR TITLE
Add unit tests for db/repos/peers - Closes #1591

### LINKED ISSUE #1591: Add unit test coverage for db/repos/peers
Parent: #1484

### Expected behavior
We need 100% coverage for db/repos/peers

### Actual behavior
Currently there is no coverage for this repo.

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?
No unit test coverage for `db/repos/peers`

### How did I fix it?
Added test cases thoroughly for every method in `db/repos/peers`. And got 100% coverage for that file.

### How to test it?
npm mocha `test/unit/db/repos/peers.js`

### Review checklist

* The PR solves #1591
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/unit/db/repos/peers.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+				yield db.peers.list();
+
+				expect(db.any.firstCall.args[0]).to.eql(peersSQL.list);
+				return expect(db.any.firstCall.args[1]).to.be.eql(undefined);
+			});
+
+			it('should return list of peers', function*() {
+				const peers = [];
+
+				for (let i = 0; i < 5; i++) {
```
Comment (vitaly-t) [inline test/unit/db/repos/peers.js]: No magic numbers, please.

File: test/unit/db/repos/peers.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+			it('should use the correct SQL with no params', function*() {
+				sinonSandbox.spy(db, 'none');
+				yield db.peers.clear();
+
+				expect(db.none.firstCall.args[0]).to.eql(peersSQL.clear);
+				return expect(db.none.firstCall.args[1]).to.be.eql(undefined);
+			});
+
+			it('should clear all peers and resolve with empty response', function*() {
+				for (let i = 0; i < 5; i++) {
```
Comment (vitaly-t) [inline test/unit/db/repos/peers.js]: Another magic number :)

File: test/unit/db/repos/peers.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+			it('should use the correct query method with correct params', function*() {
+				// Few lines are commented due to issue in pg-promise
+				// https://github.com/vitaly-t/pg-promise/issues/473
+
+				sinonSandbox.spy(db, 'none');
+				// sinonSandbox.spy(db.peers.pgp.helpers, 'insert');
+				const peer = peersFixtures.DBPeer();
+				yield db.peers.insert(peer);
+
+				return expect(db.none).to.be.calledOnce;
```
Comment (vitaly-t) [inline test/unit/db/repos/peers.js]: Insufficient test. It should attempt to locate the inserted peer instead, and see that was in fact added.

**Comment by nazarhussain** [comment]
@vitaly-t Please see the comment https://github.com/LiskHQ/lisk/pull/1641#issuecomment-369177477

**Comment by nazarhussain** [comment]
@vitaly-t Added comments to explain the seeding logic

**Comment by nazarhussain** [comment]
@vitaly-t Changes are done, please review again.

**Pull request merged**